lua/nvim-treesitter/fold.lua

```diff
@@ -45,7 +47,7 @@ local folds_levels = tsutils.memoize_by_buf_tick(function(bufnr)
     end
 
     local fold_length = stop - start + 1
-    local should_fold = fold_length >= 2
+    local should_fold = fold_length > min_fold_lines
```

Comment:
Maybe we should keep the `>= 2` too?

Reply:
does `min_fold_lines` not mean `>= min_fold_lines`? it could be `>=min(min_fold_lines, 2)`

Reply:
the default value of `foldminlines` is `1`, which is equivalent to `>= 2` here. 🤔

Do we really need to keep `>= 2` here?

Reply:
This should be `min_fold_lines + 1`

From the docs


```
With the default value of
	one a fold can only be closed if it takes up two or more screen lines
```

The default is 1, so it will work as before.

Reply:
oh, I see this was changed from `>=` to `>`.